lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -12,7 +12,15 @@ class << self
 
         sig { returns(RunnerClient) }
         def create_client
-          new
+          if File.exist?("bin/rails")
```

Comment:
This will fail in workspace that are symlinks, since the addons don't yet have access to the workspace URI announced by the editor. There's nothing to do now, but I wanted to document this.

We need to create some sort of `State` object in the Ruby LSP that addons get initialized with, containing all of the initialization options, the message queue, etc.